scripts/qa_crowbarsetup.sh

```diff
@@ -416,8 +416,10 @@ function getcloudver()
         echo -n 4
     elif [[ $cloudsource =~ ^.*(cloud|GM)5(\+up)?$ ]] ; then
         echo -n 5
-    elif [[ $cloudsource =~ ^(.+6|M[[:digit:]]+|Beta[[:digit:]]+|RC[[:digit:]]*|GMC[[:digit:]]*|GM6?(\+up)?)$ ]] ; then
+    elif [[ $cloudsource =~ ^.*(cloud|GM)6(\+up)?$ ]] ; then
         echo -n 6
+    elif [[ $cloudsource =~ ^(.+7|M[[:digit:]]+|Beta[[:digit:]]+|RC[[:digit:]]*|GMC[[:digit:]]*|GM7?(\+up)?)$ ]] ; then
+        echo -n 7
```

Comment:
this means that we cannot install any older cloud5 Milestones anymore, but if that is fine with you, still +1